CMakeModules/CompilerSettings.cmake

```diff
@@ -1,3 +1,6 @@
+# force C++11 mode
+add_definitions(-std=c++11)
```

Comment:
This causes zillions of warnings on Windows -- that should only be added for gcc & clang.